packages/next-tinacms-cloudinary/src/handlers.ts

```diff
@@ -95,9 +95,10 @@ async function listMedia(
   try {
     const {
       directory = '""',
-      limit = 500,
+      limit = '500',
```

Comment:
Why did we change this to a string?

Reply:
This is due to any overrides from the request coming in being of type `string`. I think it would be safer to be consistent.

Reply:
ahh ok - agreed!